identity-service/src/notifications/notificationProcessing.js

```diff
@@ -55,6 +57,28 @@ async function indexNotifications (notifications, tx, audiusLibs) {
       await _processCreateNotifications(audiusLibs, notif, blocknumber, timestamp, tx)
     }
   }
+  await _processSubscriberPushNotifications(tx)
+}
+
+async function _processSubscriberPushNotifications (tx) {
+  let currentTime = Date.now()
+  for (var i = 0; i < subscriberPushNotifications.length; i++) {
+    let entry = subscriberPushNotifications[i]
+    logger.debug(entry)
+    let timeSince = currentTime - entry.time
+    if (timeSince > 50000) {
```

Comment:
this value of 50s is what dictates the period during which deduping can occur...i made this up, we can change it as needed